lib/core/network/api/services/remote_config_service.dart

```diff
@@ -207,6 +209,8 @@ class RemoteConfigService {
       'signUpEnabled': true,
       "signUpLinkUrl": "https://dao.hypha.earth/hypha/login",
       'walletEnabled': false,
+      'newAccountFreshnessHours': 48,
+      'payCpuEnabledNetwork': json.encode({"telos": false, "telosTestnet": true, "eos": true, "eosTestnet": true})
```

Comment:
Don't forget to add these to firebase

Reply:
payCpuEnabledNetwork had been only in firebase, adding the other one